### PR TITLE
[pipeline] Download swss common artifact in a separated directory

### PR DESCRIPTION
**Pull request by jimmyzhai**
* Put the downloaded swss common artifact in a separated directory while build sairedis target. 
* The downloaded swss common artifact is in a sub-directory of sairedis artifact, then it avoids overriding
   the previous swss common packages.

### CODE REVIEW

File: .azure-pipelines/build-template.yml [hunk 1]
```diff
@@ -98,8 +98,10 @@ jobs:
       artifact: ${{ parameters.swss_common_artifact_name }}
       runVersion: 'latestFromBranch'
       runBranch: 'refs/heads/master'
+      path: '$(Build.SourcesDirectory)/${{ parameters.swss_common_artifact_name }}'
     displayName: "Download sonic swss common deb packages"
   - script: |
+      cd $(Build.SourcesDirectory)/${{ parameters.swss_common_artifact_name }}
       sudo dpkg -i libswsscommon_1.0.0_${{ parameters.arch }}.deb
       sudo dpkg -i libswsscommon-dev_1.0.0_${{ parameters.arch }}.deb
     workingDirectory: $(Pipeline.Workspace)
```
Comment (xumia) [inline .azure-pipelines/build-template.yml:107]: How about changing the workDirectroy to $(Build.SourcesDirectory)/${{ parameters.swss_common_artifact_name }}?

**Review by xumia — DISMISSED** [review]

**Comment by qiluo-msft** [comment]
> The sonic-sairedis artifact will not include the downloaded swss common artifact, otherwise its using
may have issue of overriding the previous swss common packages.

Then in the artifacts view, we have no way to check libswsscommon packages. This will possible limit our observation and the troubleshooting capability.

File: .azure-pipelines/build-template.yml (hunk 1 above)
Comment (jimmyzhai) [reply .azure-pipelines/build-template.yml:110]: I move the step of sairedis checkout to the first.

**Review by xumia — APPROVED** [review]

**Pull request merged**